sdk-actors/src/main/java/io/dapr/actors/runtime/ActorRuntime.java

```diff
@@ -320,10 +321,6 @@ private static DaprClient buildDaprClient(ManagedChannel channel) {
    * @return GRPC managed channel or null.
    */
   private static ManagedChannel buildManagedChannel() {
-    if (!Properties.USE_GRPC.get()) {
```

Comment:
why is this removed?

Reply:
Actually, this is needed in the case actors communicate over HTTP. Good catch.